### PR TITLE
The empire strikes back

### PR DESCRIPTION
**Pull request by rminnich**
Bring a.out format back. Go is not going to be flexible on this point.

Feel free to tell me how to help make this compile.
e.g. where is l2be again? I forget.

Signed-off-by: Ronald Minnich <rminnich@gmail.com>

### CODE REVIEW

**Comment by rminnich** [comment]
note: doesn't built.

**Comment by rminnich** [comment]
note also: intent is that sysexec will call this when it figures out it is an a.out. Lots of crap deleted since it's already done.

**Comment by rminnich** [comment]
builds.

next steps. Fill out the loading of exec
run a go binary for plan 9 which does nothing
watch it error

start filling in plan 9 system call stubs -- that can be done in syscall(), believe it or not

ratrace will be my friend

If all goes well, plan 9 go runs on harvey shortly.

File: sys/src/9/port/sysproc.c [hunk 1]
```diff
@@ -666,6 +685,8 @@ execac(Ar0* ar0, int flags, char *ufile, char **argv)
 		up->procctl = Proc_toac;
 		up->prepagemem = 1;
 	}
+	DBG("up %p: plan 9 proc\n", up);
```
Comment (gmacd) [inline sys/src/9/port/sysproc.c:688]: It's not always a plan9 proc at this point

Comment (rminnich) [reply sys/src/9/port/sysproc.c:688]: ah right I had condition DBG on plan 9. I think I can drop this print now.

Thanks!

**Pull request merged**